src/lib/operators/join_index.hpp

```diff
@@ -27,6 +27,13 @@ class JoinIndex : public AbstractJoinOperator {
 
   const std::string name() const override;
 
+  struct PerformanceData : public OperatorPerformanceData {
```

Comment:
Could we call that `Join`- or `JoinIndex`PerformanceData? `PerformanceData` inheriting from `OperatorPerformanceData` seems confusing.

Reply:
The full name is `JoinIndex::PerformanceData`, you can only use `PerformanceData` within `JoinIndex`. Did you take that into account?

Reply:
No, never mind then.